app/src/androidTest/java/com/github/orkest/ViewModel/DeezerApiIntegrationTest.kt

```diff
@@ -205,8 +255,32 @@ class DeezerApiIntegrationTest {
     fun mockFetchId(){
         val mockDeezerIntegration = DeezerApiIntegration(DeezerMockAPi.DeezerMockApiImplemented())
         val x = mockDeezerIntegration.fetchTheUserIdInTheDeezerDatabase("adadad").get()
+
+        val create = DeezerModelClasses.UserExtended("","","","","","","","","","","","","","","","","","","",
+            listOf(""),"","")
         assertEquals("Zermelo-101",x.name)
         assertEquals("2297625024",x.id)
+        assertEquals("Rocher",x.lastname)
+        assertEquals("Jean-françois",x.firstname)
+        assertEquals("jf-rocher@outlook.fr",x.email)
+        assertEquals("2",x.status)
+        assertEquals("0000-00-00",x.birthday)
+        assertEquals("2018-09-04",x.inscription_date)
+        assertEquals("",x.gender)
+        assertEquals("https://www.deezer.com/profile/2297625024",x.link)
+        assertEquals("https://api.deezer.com/user/2297625024/image",x.picture)
+        assertEquals("https://e-cdns-images.dzcdn.net/images/user/5136e5875050a362da76693a723290a0/56x56-000000-80-0-0.jpg",x.picture_small)
+        assertEquals("https://e-cdns-images.dzcdn.net/images/user/5136e5875050a362da76693a723290a0/250x250-000000-80-0-0.jpg",x.picture_medium)
+        assertEquals("https://e-cdns-images.dzcdn.net/images/user/5136e5875050a362da76693a723290a0/500x500-000000-80-0-0.jpg",x.picture_big)
+        assertEquals("https://e-cdns-images.dzcdn.net/images/user/5136e5875050a362da76693a723290a0/1000x1000-000000-80-0-0.jpg",x.picture_xl)
+        assertEquals("CH",x.country)
+        assertEquals("en",x.lang)
+        assertEquals("false",x.is_kid)
+        assertEquals("explicit_display",x.explicit_content_level)
+        assertEquals(listOf("explicit_display","explicit_no_recommendation","explicit_hide"),x.explicit_content_levels_available)
+        assertEquals("https://api.deezer.com/user/2297625024/flow",x.tracklist)
+        assertEquals("user",x.type)
+
```

Comment:
Very good tests although a lot of hardcoded values that could put in Constants or a configuration file. Also some code is duplicated you could use the setup functionnality as @Before for that

Reply:
True, will be modifie